scripts/action-handler.js

```diff
@@ -1382,29 +1380,43 @@ Hooks.once('tokenActionHudCoreApiReady', async (coreModule) => {
         // Get consume target and type
             const consumeId = item?.system?.consume?.target
             const consumeType = item?.system?.consume?.type
+            const consumeAmount = item?.system?.consume?.amount
 
-            if (consumeId === item.id) return ''
+            if (!consumeId || consumeId === item.id) return {}
 
             // Return resources
             if (consumeType === 'attribute') {
-                if (!consumeId) return ''
                 const parentId = consumeId.substr(0, consumeId.lastIndexOf('.'))
-                const target = this.actor.system[parentId]
+                const target = foundry.utils.getProperty(this.actor.system, parentId)
 
-                return (target) ? `${target.value ?? '0'}${(target.max) ? `/${target.max}` : ''}` : ''
-            }
+                if (target) {
+                    const text = `${target.value ?? '0'}${target.max ? `/${target.max}` : ''}`
+                    return {
+                        text,
+                        title : `${text} ${target.label ?? ''}`
+                    }
+                }
 
-            const target = this.items.get(consumeId)
+            } else {
 
-            // Return charges
-            if (consumeType === 'charges') {
-                const uses = target?.system.uses
+                const target = this.items.get(consumeId)
```

Comment:
```suggestion
                }
            } else {
                const target = this.items.get(consumeId)
```

Reply:
I can't tell what is being changed here and GitHub throws an error when I try to commit this suggestion. I can only assume you are removing some line breaks of empty lines around the `} else {`. Am I correct?